content/commands/info/index.md

```diff
@@ -497,13 +500,17 @@ For each database, the following line is added:
 The **debug** section contains experimental metrics, which might change or get removed in future versions.
 It won't be included when `INFO` or `INFO ALL` are called, and it is returned only when `INFO DEBUG` is used.
 
-*   `eventloop_duration_aof_sum`: Total time spent on flushing AOF in eventloop in microseconds
-*   `eventloop_duration_cron_sum`: Total time consumption of cron in microseconds (including serverCron and beforeSleep, but excluding IO and AOF flushing)
-*   `eventloop_duration_max`: The maximal time spent in a single eventloop cycle in microseconds
-*   `eventloop_cmd_per_cycle_max`: The maximal number of commands processed in a single eventloop cycle
+*   `eventloop_duration_aof_sum`: Total time spent on flushing AOF in eventloop in microseconds.
+*   `eventloop_duration_cron_sum`: Total time consumption of cron in microseconds (including serverCron and beforeSleep, but excluding IO and AOF flushing).
+*   `eventloop_duration_max`: The maximal time spent in a single eventloop cycle in microseconds.
+*   `eventloop_cmd_per_cycle_max`: The maximal number of commands processed in a single eventloop cycle.
+*   `allocator_allocated_lua`: Total bytes allocated from the allocator specifically for Lua, including internal-fragmentation.
+*   `allocator_active_lua`: Total bytes in the allocator active pages specifically for Lua, this includes external-fragmentation.
```

Comment:
Would look better as "...for Lua, including..." or "...for Lua. This includes..."
```suggestion
*   `allocator_active_lua`: Total bytes in the allocator active pages specifically for Lua, including external-fragmentation.
```